app/src/main/java/com/google/samples/apps/sunflower/adapters/GalleryAdapter.kt

```diff
@@ -41,7 +41,9 @@ class GalleryAdapter : ListAdapter<UnsplashPhoto, GalleryViewHolder>(GalleryDiff
 
     override fun onBindViewHolder(holder: GalleryViewHolder, position: Int) {
         val photo = getItem(position)
-        holder.bind(photo)
+        if (photo != null) {
```

Comment:
Paging will return null items as placeholders
We should either support them or in `PagingConfig`, disable the placeholders

Reply:
Disabled the placeholders